admin/javascript/src/silverstripe-backend.js

```diff
@@ -58,7 +58,7 @@ class SilverStripeBackend {
    * Both `payloadFormat` and `responseFormat` can use the following shortcuts for their
    * corresponding mime types:
    *
-   *   - urlencoded: application/x-www-form-url-encoded
+   *   - urlencoded: application/x-www-form-urlencoded
```

Comment:
@sminnee Just checking that you didn't write `application/x-www-form-url-encoded` on purpose? The last dash is wrong, and it causes data to be interpreted differently by CGI (doesn't end up in `$_POST`)